### DISCUSSION TITLE
is there a way to use quill.getText() in the Rich text editor package?

### DISCUSSION THREAD

**Discussion by moh-sa** [mantinedev/mantine#2150] [Q&A] [answered]
is there a way to use quill.getText() in the Rich text editor package?

**Comment by rtivital** [comment, marked as answer]
Yes, you can get quill ref – https://mantine.dev/others/rte/#get-editor-ref

**Reply by moh-sa** [reply]
like this or am i doing it wrong?

```javascript

editorRef?.current.editor.container.innerText

```

**Reply by rtivital** [reply]
I do not have much experience with quill customization, cannot advise anything besides what I've already provided

**Comment by devstc** [comment]
I know this isn't specifically what you asked, but in case it still gives you the end result you're after...

I was able to get the string that was in the editor from state.  The text in the editor is HTML and that is what is stored in state ('value' as shown in the RTE documentation example):
```
  const [value, onChange] = useState(initialValue);
  return <RichTextEditor value={value} onChange={onChange} />;
```

**Reply by moh-sa** [reply]
this way will give the HTML code, but I was looking for a way to get the text without HTML tags.

for example, using your way you will get this result: `<h2><strong>Mantine</strong> <em>Rich</em> <u>Text</u> <s>Editor</s></h2>`
but using rtivital answer, you will get this: `Mantine Rich Text Editor`
